src/views/Project/index.jsx

```diff
@@ -116,8 +118,10 @@ export default class Project extends Component {
           <Typography variant="subheading" align="center">
             Bugs & Issues
           </Typography>
+          {data && data.error && <ErrorPanel error={data.error} />}
+          {loading && <Spinner />}
+          {issues && <TasksTable items={issues} />}
```

Comment:
We probably don't want to show `TaskTable` in its empty state if it's still loading. Also, now that issues defaults to `[]` when empty, we can omit it from here.

```jsx
{!loading && <TasksTable items={issues} />}
```